src/Configuration/Configuration.php

```diff
@@ -210,9 +210,9 @@ public function getTestFrameworkOptions(): ?string
         return $this->testFrameworkOptions;
     }
 
-    public function getExistingCoveragePath(): ?string
```

Comment:
I would love to retain this name as it is clear that we are working with the path to coverage provided by user, hence existing.

Probably `getExistingCoverageBasePath()`?

`getCoverageBasePath()` is too common and easy to mix up with another `retrieveCoverageBasePath()` method.

Reply:
👍